src/assets/patterns/molecules/artboards/css/artboards.css

```
@@ -76,8 +76,8 @@
 }
 
 .m-artboards__list {
-    flex: 1 0;
-    overflow: auto;
+    flex: 1 1;
+    overflow: scroll;
```

```suggestion
    overflow-y: scroll;
```
If we restrict the CSS property to the y axis we can prevent the horizontal scrollbar. What do you think?